qualification/Jenkinsfile

```
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2021-2022, National Research Foundation (SARAO)
+ *
+ * Licensed under the BSD 3-Clause License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ *   https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+/* This file tells Jenkins how to execute the qualification tests for this repo.
+ * Everything runs in docker containers, so in theory any Jenkins server should
+ * be able to parse this file, however the pytest-jenkins.ini file needs to be
+ * configured for the hardware it is running on. The Docker Engine being used needs
+ * to have been configured to use the NVIDIA Container Runtime and the node which the
+ * container runs on needs the Nvidia Driver installed.
```

```suggestion
 * to have been configured to use the NVIDIA Container Runtime and the node which the
 * container runs on needs the NVIDIA OFED drivers installed.
```
to clarify that it's the OFED (networking) drivers, not GPU drivers.